assets/src/hooks/useVehicleAndRouteForNotification.ts

```diff
@@ -33,15 +33,18 @@ const parseVehicleOrGhostAndRouteData = ({
 }
 
 const useVehicleAndRouteForNotification = (
-  selectedTripIdsForNotification?: TripId[]
+  notification?: Notification
 ): VehicleOrGhostAndRoute | undefined => {
```

Comment:
This would need to change to `: VehicleOrGhostAndRoute | null | undefined` to be able to use it to detect when it fails instead of using the separate context for that.